src/utils/openmc/material.py

```diff
@@ -76,7 +76,9 @@ def set_id(self, material_id=None):
 
         # If the Material already has an ID, remove it from global list
         if not self._id is None:
-            MATERIAL_IDS.remove(self._id)
+            try:
```

Comment:
If i remember correctly that is what I had been doing at one point, but now that I look through the cleaned-up BEAVRS scripts again it seems we no longer are doing that (and shouldn't be - set_id should always be used, I beleive).  I'm thinking I'll actually remove this change.